DirectXTex/DirectXTexDDS.cpp

```diff
@@ -191,74 +191,92 @@ namespace
 
         constexpr size_t MAP_SIZE = sizeof(g_LegacyDDSMap) / sizeof(LegacyDDS);
         size_t index = 0;
-        for (index = 0; index < MAP_SIZE; ++index)
+        // Unreal Tournament 2004 DDS files use a incorrectly populated DDS_PIXELFORMAT
```

Comment:
I think removing the specific reference to UT2004 from here and making it the following is clearer for the long-term since it may apply to other DDS files as well.

```
if (ddpf.size == 0 && ddpf.flags == 0 && ddpf.fourCC != 0)
{
    // Handle some DDS files where the DDPF_PIXELFORMAT is mostly zero
```